packages/react/src/Token/IssueLabelToken.module.css

```diff
@@ -51,10 +52,8 @@
 }
 
 /* Selected state */
-.IssueLabel:where(
-    [data-selected='true'][data-in-color-mode*='light'],
-    [data-selected='true'][data-in-color-mode='auto']
-  ) {
+[data-color-mode*='light'] .IssueLabel:where([data-selected='true']),
+[data-color-mode='auto'] .IssueLabel:where([data-selected='true']) {
```

Comment:
auto should be handled like this:

https://github.com/primer/react/blob/a15faefa58c20742557de7e95bfe2813794e93a7/packages/react/src/BaseStyles.module.css#L28-L38